modules/dkan/dkan_periodic_updates/dkan_periodic_updates.module

```diff
@@ -408,6 +416,91 @@ function dkan_periodic_updates_state() {
   return $output;
 }
 
+
+/**
+ * Page callback for periodic updates reset page.
+ *
+ * @param $form
+ * @param $form_state
+ *
+ * @return mixed
+ */
+function dkan_periodic_updates_reset_form($form, &$form_state) {
+  $active = variable_get('dkan_periodic_updates_status');
+  if ($active) {
+    $file = variable_get('dkan_periodic_updates_manifest');
+    $file = file_load($file);
+    if ($file) {
+      $handle = fopen($file->uri, 'r');
+      $manifest_options = [];
+      $headers = fgetcsv($handle, 0, ',');
+      while (($data = fgetcsv($handle, 0, ",")) !== FALSE) {
+        $resource_data = array_combine($headers, $data);
+        if (!empty(variable_get('dkan_periodic_updates_' . $resource_data['resource_id'], ''))) {
+          $manifest_options[$resource_data['resource_id']] = $resource_data['file_url'];
+        }
+      }
+      fclose($handle);
+
+      $form['resource_id'] = [
+        '#type' => 'select',
+        '#empty_option' => t('Select an option'),
+        '#title' => t('Resource'),
+        '#options' => $manifest_options,
+        '#description' => t('Select the resource to force import in the next cron execution.'),
+      ];
+
+      $form['reset'] = [
+        '#type' => 'submit',
+        '#value' => t('Reset'),
+      ];
+
+    }
+    else {
+      $form['state'] = [
+        '#markup' => '<p class="alert alert-warning">' . t('No manifest was found.') . '</p>',
+      ];
+    }
+  }
+  else {
+    $form['state'] = [
+      '#markup' => '<p class="alert alert-warning">' . t('Periodic updates are disabled.') . '</p>',
+    ];
+  }
+
+  return $form;
+}
+
+/**
+ * Validate function for periodic updates reset form.
+ *
+ * @param $form
+ * @param $form_state
+ */
+function dkan_periodic_updates_reset_form_validate($form, &$form_state) {
+  if (empty($form_state['values']['resource_id'])) {
+    form_set_error('resource_id', t('You must select a resource'));
+  }
+  if (empty(variable_get('dkan_periodic_updates_' . $form_state['values']['resource_id'], ''))) {
+    form_set_error('resource_id', t('The resource has not been update'));
+  }
+}
+
+/**
+ * Submit function for periodic updates reset form.
+ *
+ * @param $form
+ * @param $form_state
+ */
+function dkan_periodic_updates_reset_form_submit($form, &$form_state) {
+  if (empty($form_state['values']['resource_id'])) {
+    form_set_error('resource_id', t('You must select a resource'));
+  }
+  variable_set('dkan_periodic_updates_' . $form_state['values']['resource_id'], '');
```

Comment:
As filling the field is mandatory, I think we should add the required indication to it.

Reply:
Change included